content/ops/creating-a-local-dev-environment-in-Virtual-Box.md

```diff
@@ -196,6 +196,8 @@ Secure copy the cflinuxfs2 container to the CF VM.
 
 Connect using the Cloud Foundry CLI.
 
+        # connect back to the bosh-lite VM instance
+        vagrant ssh
```

Comment:
Looks like this isn't indented the same as the commands below? Causing a formatting problem.